modules/administration-guide/pages/devworkspace-operator.adoc

```diff
@@ -5,22 +5,163 @@
+|Scope |Applies to the DWO itself and all {devworkspace}s managed by DWO |Applies to {devworkspace}s created by {prod-short}
+|Precedence |Overridden by fields set in {prod-short}-owned config |Takes precedence over global config if both define the same field
+|Primary use case |Used to define default, broad settings that apply to DWO in general. |Used to define specific configuration for {devworkspace}s created by {prod-short}
+|===
+
+For example, by default {prod-short} creates and manages a non-global DWOC in the {prod-short} namespace named `devworkspace-config`. This DWOC contains configuration specific to {prod-short} workspaces, and is maintained by {prod-short} depending on how the CheCluster CR is configured. When {prod-short} creates a workspace, {prod-short} will add a reference to the {prod-short}-owned DWOC via the `controller.devfile.io/devworkspace-config` attribute.
+
+.Example of {devworkspace} configuration attribute
+image::devworkspace-operator/devworkspace-config-attribute.png[DevWorkspace config attribute example,width=624,height=280]
+
+.DevWorkspaceRouting
+
+The DevWorkspaceRouting custom resource defines details about the endpoints of a {devworkspace}. Every {devworkspace} will have its corresponding DevWorkspaceRouting object that specifies the workspace's container endpoints. Endpoints defined from the devfile, as well as endpoints defined by the editor definition will appear in the DevWorkspaceRouting custom resource.
+
+[source,yaml]
+----
+apiVersion: controller.devfile.io/v1alpha1
+kind: DevWorkspaceRouting
+metadata:
+  annotations:
+    controller.devfile.io/devworkspace-started: 'false'
+  name: routing-workspaceb14aa33254674065
+  labels:
+    controller.devfile.io/devworkspace_id: workspaceb14aa33254674065
+spec:
+  devworkspaceId: workspaceb14aa33254674065
+  endpoints:
+    universal-developer-image:
+      - attributes:
+          cookiesAuthEnabled: true
+          discoverable: false
+          type: main
+          urlRewriteSupported: true
+        exposure: public
+        name: che-code
+        protocol: https
+        secure: true
+        targetPort: 3100
+  podSelector:
+    controller.devfile.io/devworkspace_id: workspaceb14aa33254674065
+  routingClass: che
+status:
+  exposedEndpoints:
+    ...
+----
+
+.{devworkspace} Operator's operands
+
+The {devworkspace} Operator has two operands, the controller deployment and the webhook deployment.
+
+[source,terminal]
+----
+$ oc get pods -l 'app.kubernetes.io/part-of=devworkspace-operator' -o custom-columns=NAME:.metadata.name -n openshift-operators
+NAME
+devworkspace-controller-manager-66c6f674f5-l7rhj <1>
+devworkspace-webhook-server-d4958d9cd-gh7vr <2>
+devworkspace-webhook-server-d4958d9cd-rfvj6 <2>
+----
+
+<1> The {devworkspace} controller pod, which is responsible for reconciling custom resources
+<2> The {devworkspace} operator webhook server pods
+
+=== Configuring the {devworkspace}-controller-manager deployment
```

Comment:
Shouldn't this be `h2` ?
```suggestion
== Configuring the {devworkspace}-controller-manager deployment
```

Reply:
Yes, it should be h2, @deerskindoll 's suggestion fixes it (it seems we cannot use `==`, we must use `.`)